pom.xml

```diff
@@ -350,11 +350,6 @@
                 <artifactId>joda-time</artifactId>
                 <version>2.3</version>
             </dependency>
```

Comment:
joda still made it in :)

Reply:
is that because we just want to manage it?

Reply:
yeah, i'll add a comment - version 2.3 is pulled in by scalatra (we should probably upgrade scalatra too), and 2.7 is pulled in by osm4j. didn't want to file a new cq, so...

Reply:
sounds good. Yeah I think that some scalatra upgrades could be good for 1.4.0 maybe actually a version review of a few things...I was specifically looking at 2.11.8 bc of the repl tab completion improvements but that's more stuff I guess...also tracks the spark version.